core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorTableHandle.java

```diff
@@ -13,6 +13,10 @@
  */
 package io.trino.spi.connector;
 
+/**
+ * A handle to a table returned from the connector to the engine.
+ * It will be used by the engine whenever given table will be accessed.
```

Comment:
It's worth noting whether `ConnectorTableHandle` can refer to a relation (a view or a materialized view), or only tables.